src/reducers/productReducer.js

```diff
@@ -0,0 +1,65 @@
+import productService from '../services/productService';
+
+export const productActions = {
+    SET_GETTING_PRODUCTS: 'GETTING_PRODUCTS',
+    SET_PRODUCTS: 'SET_PRODUCTS',
+    SET_FILTERED_PRODUCTS: 'SET_FILTERED_PRODUCTS'
+};
+
+export const initialState = {
+    products: [],
+    filteredProducts: [],
```

Comment:
Mun mielestä filtteröityjä tuotteita ei tarvitse erikseen tallentaa stateen, koska siellä listauskomponentissa voidaan suorittaa filter noihin tuotteisiin. Jos products muuttuu, niin tällöin filteredProducts ei välttämättä pysy menossa mukana